.github/workflows/coverage.yaml

```diff
@@ -110,60 +110,8 @@ jobs:
           path: coverage/playwright-report
           retention-days: 1
 
-  publish-coverage-report:
-      name: publish-coverage-report
-      runs-on: ubuntu-latest
-      needs: e2e-coverage
-      continue-on-error: true
-      steps:
-        - uses: actions/checkout@v4
-          with:
-            ref: gh-pages
-            token: ${{ secrets.GITHUB_TOKEN }}
-        - name: Cleanup coverage directory
-          run: |
-            rm -rf coverage
-            mkdir coverage
-        - name: Download coverage report artifact
-          uses: actions/download-artifact@v4
-          with:
-            name: lxd-coverage
-            path: coverage
-        # user git configs are needed for git commands to work
-        # actual authentication is done using secrets.GITHUB_TOKEN with write permission
-        - name: Set Git User
-          run: |
-            git config --global user.email "github-action@example.com"
-            git config --global user.name "GitHub Action"
-        - name: Push coverage Report
-          timeout-minutes: 3
-          run: |
-            git add .
-            git commit -m "workflow: update coverage report"
-            
-            # In case of another action job pushing to gh-pages while we are rebasing for the current job
-            while true; do
-              git pull --rebase
-              if [ $? -ne 0 ]; then
-                echo "Failed to rebase. Please review manually."
-                exit 1
-              fi
-  
-              git push
-              if [ $? -eq 0 ]; then
-                echo "Successfully pushed HTML report to repo."
-                exit 0
-              fi
-            done
-        - name: Output Report URL as Worfklow Annotation
-          run: |
-            FULL_HTML_REPORT_URL=https://canonical.github.io/lxd-ui/coverage
-            echo "::notice title=Published Playwright Test Report::$FULL_HTML_REPORT_URL"
-
-
   tics-report:
     runs-on: ubuntu-latest
-    needs: publish-coverage-report
```

Comment:
@edlerd not sure why `tics-report` depended on `publish-coverage-report`. From what I can see, just removing this line in this job should be fine, let me know if I missed something here